### PR TITLE
Add Lighthouse PWA scores query

### PR DESCRIPTION
**Pull request by tomayac**
Breaking down https://github.com/HTTPArchive/bigquery/pull/47 into small PRs, this is the first.

### CODE REVIEW

**Comment by tomayac** [comment]
This is dependent on https://github.com/HTTPArchive/httparchive.org/pull/74.

File: sql/timeseries/pwaScores.sql [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  OFFSET
+    (51)], 2) AS p50,
+  ROUND(APPROX_QUANTILES(lighthouse_pwa_score, 101)[
+  OFFSET
+    (76)], 2) AS p75,
+  ROUND(APPROX_QUANTILES(lighthouse_pwa_score, 101)[
+  OFFSET
+    (91)], 2) AS p90
+FROM (
+  SELECT
```
Comment (rviscomi) [inline sql/timeseries/pwaScores.sql]: Can you rewrite this without the nested query? Eg https://github.com/HTTPArchive/bigquery/blob/master/sql/timeseries/responsiveImages.sql

You'll be duplicating the UDF call to getPWAScore but I think it's ok.

File: sql/timeseries/pwaScores.sql [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    getPWAScore(report) AS lighthouse_pwa_score,
+    REGEXP_EXTRACT(_TABLE_SUFFIX, "\\d{4}(?:_\\d{2}){2}") AS date,
+    REGEXP_EXTRACT(_TABLE_SUFFIX, ".*_(\\w+)$") AS client
```
Comment (rviscomi) [inline sql/timeseries/pwaScores.sql]: Let's be consistent with how we extract the date and client fields. Can you rewrite this to use the `SUBSTR` and `ENDS_WITH` approaches in the responsiveImages query above?

File: sql/timeseries/pwaScores.sql [hunk 3]
```diff
@@ -0,0 +1,44 @@
+#standardSQL
+CREATE TEMPORARY FUNCTION
```
Comment (rviscomi) [inline sql/timeseries/pwaScores.sql]: I find BigQuery's "format query" function generates some pretty ugly formatting. Could you copy the style from this [crux query](https://github.com/HTTPArchive/bigquery/blob/master/sql/histograms/cruxFcp.sql) with a UDF?

Another thing to reformat, keep the fields on one line, eg

```sql
ROUND(APPROX_QUANTILES(lighthouse_pwa_score, 101)[OFFSET(11)], 2) AS p10,
```

**Comment by tomayac** [comment]
@rviscomi I have addressed your comments, PTAL!

File: sql/timeseries/pwaScores.sql (hunk 1 above)
Comment (tomayac) [reply sql/timeseries/pwaScores.sql]: Sounds good, changed.

File: sql/timeseries/pwaScores.sql (hunk 2 above)
Comment (tomayac) [reply sql/timeseries/pwaScores.sql]: +1 for consistency, changed to follow your style.

File: sql/timeseries/pwaScores.sql (hunk 3 above)
Comment (tomayac) [reply sql/timeseries/pwaScores.sql]: I fully agree about the ugly formatting, just thought we settled for whatever it produces. Have changed to a more sane style manually now. (FYI, I have opened an issue with the formatter internally.)

**Pull request merged**